ballerina/tests/client_endpoint_test.bal

```diff
@@ -216,7 +216,7 @@ public function testPutFileContent() returns error? {
 
     Error? response = clientEp->put(newFilePath, bStream);
     if (response is Error) {
-        log:printError("Error in put operation", 'error = response);
+        test:assertFail(msg = "Found unexpected response type from `put` operation");
```

Comment:
Do we need to have this if condition cos in this scenario we should fail the test case if there is an error. And sicne we have added `returns error?` to the test method signature we could just use following;
```ballerina
check clientEp->put(newFilePath, bStream);
```
Please update in other places where this could be relevant.

Reply:
Added the error logging to log the error message as well, in order to improve the error meaning. It would make easier to find the root cause when tests fail.